crates/neon/src/sys/bindings/functions.rs

```diff
@@ -65,6 +65,14 @@ mod napi1 {
                 result: *mut usize,
             ) -> Status;
 
+            fn get_value_string_utf16(
+                env: Env,
+                value: Value,
+                buf: *mut u16,
```

Comment:
I'm not super strong in C++. Is `char16_t` practically always `u16`? The spec seems to define it as "at _least_ a u16." It looks like `rust-bindgen` unconditionally makes it a `u16`.

Reply:
I was worried about that one too. Thought maybe even the `c_short` type is a better fit. As you said, [rust-bindgen](https://github.com/rust-lang/rust-bindgen/blob/678daf56a2384252304fc27ed2b46e769da1890f/bindgen/ir/int.rs#L57) defines it as  `u16` as well, so I'd say it's correct 🤔

Reply:
Looking around, it seems like this is always a `char16_t` and Node itself doesn't support a lot of esoteric CPU architectures, so I think this is fine.

Do you mind adding a comment here as a hint in case it *does* cause a problem at some point?